plugins/modules/notification/slack.py

```diff
@@ -100,7 +100,11 @@
   attachments:
     description:
       - Define a list of attachments. This list mirrors the Slack JSON API.
-      - For more information, see also in the (U(https://api.slack.com/docs/attachments)).
+      - For more information, see U(https://api.slack.com/docs/attachments).
+  blocks:
+    description:
+      - Define a list of blocks. This list mirrors the Slack JSON API.
+      - For more information, see U(https://api.slack.com/block-kit).
```

Comment:
```suggestion
      - For more information, see U(https://api.slack.com/block-kit).
    type: list
    elements: dict
    version_added: 1.0.0
```

Reply:
Should this change – minus the version number – not be made to every param?

Reply:
Yep, but as I wrote, in another PR. These currently do not result in errors because these errors are ignored for this plugin: https://github.com/ansible-collections/community.general/blob/main/tests/sanity/ignore-2.11.txt#L951-L953

It's probably best to remove these ignore.txt lines (and the similar ones in the othe rignore-2.x.txt files) and fix things until everything's fine.